### DISCUSSION TITLE
Allow for me to release new versions

### DISCUSSION THREAD

**Discussion by pftg** [donv/capybara-screenshot-diff#76] [General]
Hey @donv @UweKubosch we have not released latest refactoring, is it ok for me to create new releases?

**Comment by donv** [comment]
@pftg Yes, please go ahead.

**Reply by pftg** [reply]
Could you add me https://rubygems.org/profiles/jetthoughts on rubygems

**Reply by donv** [reply]
Done.  Please release responsibly 😄
